pulp_python/app/viewsets.py

```diff
@@ -1,22 +1,31 @@
 from pulpcore.plugin import viewsets as platform
+from django_filters.rest_framework import filterset
+from django_filters.filters import BaseInFilter
 
 from . import models, serializers
 
 
-class PythonContentViewSet(platform.ContentViewSet):
+class PythonPackageContentFilter(filterset.FilterSet):
     """
-    A ViewSet for PythonContent.
+    Filter for PythonPackageContent
+    """
+
+    classifier = BaseInFilter(name='classifier__name', lookup_expr='in')
```

Comment:
I don't think "in" is what users would want. ?classifier=Topic :: Scientific/Engineering,Topic :: Software Development (except properly escaped) would return an *or* of those tags. 

However we do it, I think we should add filtering in a separate story.